ENIGMAsystem/SHELL/Universal_System/Resources/fontinit.cpp

```diff
@@ -138,11 +138,10 @@ void exe_loadfonts(FILE* exe) {
     font.height = ymax - ymin + 2;
     font.yoffset = -ymin + 1;
 
-    font.texture = graphics_create_texture(twid, thgt, twid, thgt, pixels, false);
+    font.texture = graphics_create_texture(RawImage((unsigned char*)pixels, twid, thgt), false);
     font.twid = twid;
     font.thgt = thgt;
 
-    delete[] pixels;
```

Comment:
@fundies This isn't good because the size of `pixels` here was `size + 1` where size may not have been the product of width and height of the texture. Why did you remove so many of these deletes? Did you make RawImage handle the cleanup?

Reply:
Yes RawImage automatically deletes when it goes out of scope in it's destructor

Reply:
Hrm, font is tricky edge case then, I wonder what the extra +1 on the size is.

Reply:
I would rather 3 extra lines than some confusing hack though